content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Gateway API
+content_type: concept
+description: >-
+  Gateway API is a family of API kinds that provide dynamic infrastructure provisioning
+  and advanced traffic routing.
+weight: 55
+---
+
+<!-- overview -->
+
+Make network services available by using an extensible, role-oriented, protocol-aware configuration
+mechanism. The [Gateway API](https://gateway-api.sigs.k8s.io/) is an {{<glossary_tooltip text="add-on" term_id="addons">}}
+containing API [kinds](https://gateway-api.sigs.k8s.io/references/spec/) that provide dynamic infrastructure
+provisioning and advanced traffic routing.
+
+<!-- body -->
+
+## Design principles
+
+The following principles shaped the design and architecture of the Gateway API:
+
+* __Role-oriented:__ Gateway API kinds are modeled after organizational roles that are
+  responsible for managing Kubernetes service networking:
+  * __Infrastructure Provider:__ Manages infrastructure that allows multiple isolated clusters
+    to serve multiple tenants, e.g. a cloud provider.
+  * __Cluster Operator:__ Manages clusters and is typically concerned with policies, network
+    access, application permissions, etc.
+  * __Application Developer:__ Manages an application running in a cluster and is typically
+    concerned with application-level configuration and [Service](/docs/concepts/services-networking/service/)
+    composition.
+* __Portable:__ Gateway API specifications are defined as [custom resources](docs/concepts/extend-kubernetes/api-extension/custom-resources)
+  and are supported by many [implementations](https://gateway-api.sigs.k8s.io/implementations/).
+* __Expressive:__ Gateway API kinds support functionality for common traffic routing use cases
+  such as header-based matching, traffic weighting, and others that were only possible in
+  [Ingress](/docs/concepts/services-networking/ingress/) by using custom annotations.
+* __Extensible:__ Gateway allows for custom resources to be linked at various layers of the API.
+  This makes granular customization possible at the appropriate places within the API structure.
+
+## Resource model
+
+The Gateway API has three stable API kinds:
+
+* __GatewayClass:__ Defines a set of gateways with common configuration and managed by a controller
+  that implements the class.
+
+* __Gateway:__ Defines an instance of traffic handling infrastructure, such as cloud load balancer.
+
+* Route kinds: The Gateway API specifies API kinds, HTTPRoute for example, that define protocol-specific
```

Comment:
I like mentioning that there are other routes but I think we should either make `HTTPRoute` more prominent here, given that it alone has its own section later on. OR rename the later section "Route kinds" and mention how the other route kinds fit in. As is, it's weird that HTTPRoute is "just an example" of presumably a number of routes here but later on is the only one we discuss without mention of what the status/place of the other routes is.

Reply:
I have updated this section to use HTTPRoute instead of Route kinds based on your feedback. I think this makes sense since this section is only talking about the GA kinds. I include a link in the what's next to reference the additional kinds.